<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PrimaryLogOptionsWidget</class>
 <widget class="QWidget" name="PrimaryLogOptionsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>601</width>
    <height>311</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>DejaVu Sans</family>
    <pointsize>8</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Primary log</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_25">
   <property name="spacing">
    <number>2</number>
   </property>
   <property name="leftMargin">
    <number>4</number>
   </property>
   <property name="topMargin">
    <number>4</number>
   </property>
   <property name="rightMargin">
    <number>4</number>
   </property>
   <property name="bottomMargin">
    <number>4</number>
   </property>
   <item>
    <widget class="QFrame" name="frmPrimaryLog">
     <property name="autoFillBackground">
      <bool>true</bool>
     </property>
     <property name="frameShape">
      <enum>QFrame::Panel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_5">
      <property name="topMargin">
       <number>4</number>
      </property>
      <property name="bottomMargin">
       <number>4</number>
      </property>
      <property name="verticalSpacing">
       <number>2</number>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="lblPrimaryLog">
        <property name="text">
         <string>Primary log:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="cmbPrimaryLogType">
        <item>
         <property name="text">
          <string>None</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>File</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Database</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="Line" name="line_7">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QComboBox" name="cmbPrimaryLogMode">
        <item>
         <property name="text">
          <string>save information about every performed test (Full)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>write a record when test status changes (Brief)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>write a record when test status changes + midnight logging (Midnight)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>write a record when test status or reply value changes (Reply)</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="lblPrimaryLogMode">
        <property name="text">
         <string>Default logging mode:</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QStackedWidget" name="stwPrimaryLog">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="frameShape">
      <enum>QFrame::Panel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="pagePrimaryLogFile">
      <layout class="QHBoxLayout" name="horizontalLayout_19">
       <item>
        <widget class="QGroupBox" name="boxPrimaryLogFile">
         <property name="title">
          <string> File log  </string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_23">
          <item>
           <widget class="QComboBox" name="cmbPrimaryLogFile">
            <item>
             <property name="text">
              <string>Log to specific file</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Automatically change the log every month</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Automatically change the log every week</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Automatically change the log every day</string>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_18">
            <property name="spacing">
             <number>0</number>
            </property>
            <item>
             <widget class="QLineEdit" name="editPrimaryLogFileName"/>
            </item>
            <item>
             <widget class="QToolButton" name="btnPrimaryLogDir">
              <property name="text">
               <string>...</string>
              </property>
              <property name="icon">
               <iconset resource="../../qHostMonitor.qrc">
                <normaloff>:/img/open.png</normaloff>:/img/open.png</iconset>
              </property>
              <property name="iconSize">
               <size>
                <width>16</width>
                <height>16</height>
               </size>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="boxPrimaryLogFormat">
         <property name="title">
          <string> Log format  </string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_22">
          <property name="spacing">
           <number>2</number>
          </property>
          <property name="leftMargin">
           <number>4</number>
          </property>
          <property name="topMargin">
           <number>4</number>
          </property>
          <property name="rightMargin">
           <number>4</number>
          </property>
          <property name="bottomMargin">
           <number>4</number>
          </property>
          <item>
           <widget class="QRadioButton" name="rbPrimaryLogHTML">
            <property name="text">
             <string>HTML file</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="rbPrimaryLogText">
            <property name="text">
             <string>Text file</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="rbPrimaryLogDBF">
            <property name="text">
             <string>DBF file</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="pagePrimaryLogODBC">
      <layout class="QHBoxLayout" name="horizontalLayout_23">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>2</number>
       </property>
       <property name="topMargin">
        <number>2</number>
       </property>
       <property name="rightMargin">
        <number>2</number>
       </property>
       <property name="bottomMargin">
        <number>2</number>
       </property>
       <item>
        <widget class="QGroupBox" name="boxPrimaryLogODBC">
         <property name="autoFillBackground">
          <bool>true</bool>
         </property>
         <property name="title">
          <string> ODBC log  </string>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_22">
          <property name="spacing">
           <number>2</number>
          </property>
          <property name="leftMargin">
           <number>2</number>
          </property>
          <property name="topMargin">
           <number>2</number>
          </property>
          <property name="rightMargin">
           <number>2</number>
          </property>
          <property name="bottomMargin">
           <number>2</number>
          </property>
          <item>
           <layout class="QFormLayout" name="formLayout_2">
            <property name="horizontalSpacing">
             <number>2</number>
            </property>
            <property name="verticalSpacing">
             <number>2</number>
            </property>
            <item row="0" column="0">
             <widget class="QLabel" name="lblPrimaryLogDataSource">
              <property name="text">
               <string>Data source</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QComboBox" name="cmbPrimaryLogDatasource">
              <property name="editable">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <layout class="QVBoxLayout" name="verticalLayout_24">
              <property name="spacing">
               <number>2</number>
              </property>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_21">
                <property name="spacing">
                 <number>2</number>
                </property>
                <item>
                 <widget class="QLabel" name="lblPrimaryLogSQL">
                  <property name="text">
                   <string>SQL query</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QToolButton" name="btnPrimaryLogSqlEdit">
                  <property name="maximumSize">
                   <size>
                    <width>24</width>
                    <height>24</height>
                   </size>
                  </property>
                  <property name="text">
                   <string>...</string>
                  </property>
                  <property name="icon">
                   <iconset resource="../../qHostMonitor.qrc">
                    <normaloff>:/img/loupe.png</normaloff>:/img/loupe.png</iconset>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <spacer name="verticalSpacer_3">
                <property name="orientation">
                 <enum>Qt::Vertical</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>20</width>
                  <height>40</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QPushButton" name="btnPrimaryLogOdbcTest">
                <property name="text">
                 <string>Test</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item row="1" column="1">
             <widget class="QPlainTextEdit" name="plainPrimaryLogSQL">
              <property name="verticalScrollBarPolicy">
               <enum>Qt::ScrollBarAlwaysOn</enum>
              </property>
              <property name="plainText">
               <string>Insert into hmlog (eventtime, testname, status, reply, testid, testmethod) VALUES ('%DateTime%', '%TestName%', '%Status%', '%Reply%', %TestID%, '%TestMethod%')</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QFormLayout" name="formLayout">
            <property name="horizontalSpacing">
             <number>4</number>
            </property>
            <property name="verticalSpacing">
             <number>4</number>
            </property>
            <item row="0" column="0">
             <widget class="QLabel" name="lblPrimaryLogLogin">
              <property name="text">
               <string>Login</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLineEdit" name="editPrimaryLogLogin"/>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="lblPrimaryLogPassword">
              <property name="text">
               <string>Password</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QLineEdit" name="editPrimaryLogPassword"/>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="lblPrimaryLogTimeout">
              <property name="text">
               <string>Timeout</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <layout class="QHBoxLayout" name="horizontalLayout_20">
              <item>
               <widget class="QSpinBox" name="spinPrimaryLogTimeout">
                <property name="value">
                 <number>10</number>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="lblPrimaryLogTimeoutSec">
                <property name="text">
                 <string>sec</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frmPrimaryLogAlerts">
     <property name="frameShape">
      <enum>QFrame::Panel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_6">
      <property name="leftMargin">
       <number>2</number>
      </property>
      <property name="topMargin">
       <number>2</number>
      </property>
      <property name="rightMargin">
       <number>2</number>
      </property>
      <property name="bottomMargin">
       <number>2</number>
      </property>
      <property name="spacing">
       <number>2</number>
      </property>
      <item row="0" column="0">
       <widget class="QCheckBox" name="chkPrimaryLogAlertInaccessible">
        <property name="text">
         <string>Execute alert profile when log inaccessible</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QComboBox" name="cmbPrimaryLogAlertAlive"/>
      </item>
      <item row="3" column="0">
       <widget class="QCheckBox" name="chkPrimaryLogAlertAlive">
        <property name="text">
         <string>Execute alert profile when log comes to life</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="cmbPrimaryLogAlertInaccessible">
        <property name="editable">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QToolButton" name="btnPrimaryLogAlertInaccessible">
        <property name="maximumSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
        <property name="text">
         <string>Alert</string>
        </property>
        <property name="icon">
         <iconset resource="../../qHostMonitor.qrc">
          <normaloff>:/img/foldersTree.png</normaloff>:/img/foldersTree.png</iconset>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QToolButton" name="btnPrimaryLogAlertAlive">
        <property name="maximumSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
        <property name="text">
         <string>Alert</string>
        </property>
        <property name="icon">
         <iconset resource="../../qHostMonitor.qrc">
          <normaloff>:/img/foldersTree.png</normaloff>:/img/foldersTree.png</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../qHostMonitor.qrc"/>
  <include location="../../qHostMonitor.qrc"/>
 </resources>
 <connections/>
</ui>
